<h3>BSc Biomedicine Graduate @ UEA</h3>
<h4>Areas of interest: Bioinformatics, Data Science</h4>
<dl>
  <dt>Final year project: "Comparision of RNA-seq methodologies, the impact on differential gene expression (2022)"</dt>
  <dd>- Comparison of traditional read-mapping pipeline on D.<i> melanogaster</i> <a href="https://github.com/mikelove/DESeq2">DESeq2</a> against k-mer pseudoalignment method <a href="https://github.com/pachterlab/kallisto">Kallisto</a> for measuring sex-specific post-mating transcriptional responses.</dd>
  <dd>- Synthesis: Use Kallisto! No reference .bam required, low computational cost and alligns faster with no significant accuracy loss.
  <dd>- Languages used: Python (pandas/polars, fastqc, matplotlib, seaborn), BASH, R (count normalisation and heatmap generation).</dd>
</dl>
